makedebian.pl

```diff
@@ -260,11 +260,17 @@
 open(SCRIPT, ">$postinstall_file");
 print SCRIPT <<EOF;
 #!/bin/sh
+justinstalled=1
```

Comment:
Is justinstalled set to 1 when Webmin is installed for the first time?

Reply:
Rather `justinstalled` is unset the moment after it's set, in case `/etc/webmin` exists.

Reply:
Ok, but what it's detecting is whether webmin is being installed for the first time?

Reply:
Yes, and it's serving just a simple purpose to know whether to run `/etc/webmin/start` or `/etc/webmin/.post-install` (please don't forget that `../start` and `../.post-install` files are different on `systemd` systems).

Reply:
Got it, makes sense

Reply:
.. i.e. in case of older systems, it wouldn't matter, and in practice `../start` would be called in both cases (install/upgrade) .. but on `systemd` it's different.